packages/models-library/src/models_library/api_schemas_webserver/workspaces.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from typing import NamedTuple
+
+from models_library.basic_types import IDStr
+from models_library.users import GroupID
+from models_library.workspaces import WorkspaceID
+from pydantic import Extra, PositiveInt
+
+from ..access_rights import AccessRights
+from ._base import InputSchema, OutputSchema
+
+
+class WorkspaceGet(OutputSchema):
+    workspace_id: WorkspaceID
+    name: str
+    description: str | None
+    thumbnail: str | None
+    created_at: datetime
+    modified_at: datetime
+    owner_primary_gid: GroupID
+    my_access_rights: AccessRights
+
+
+class WorkspaceGetPage(NamedTuple):
+    items: list[WorkspaceGet]
+    total: PositiveInt
+
+
+class CreateWorkspaceBodyParams(InputSchema):
+    name: str
+    description: str | None = None
```

Comment:
same here in general. No documentation, how do you define these in the openAPI?

Reply:
name, description and thumbnail I guess are obvious :+1: